tests/check_data.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import pandas as pd
+import numpy as np
+
+
+def check_patient_data(file, mode):
+    d = pd.read_csv(file)
+    if mode == "pediatrics":
+        d.rename(columns={"agedays": "age"}, inplace=True)
+    elif mode == "adults":
+        d.rename(columns={"age_years": "age"}, inplace=True)
+    else:
+        raise ValueError("mode must be 'adults' or 'pediatrics")
+
+    warnings_list = []
+    if d["sex"].isin([0, 1]).all() == False:
```

Comment:
Not required to change, but fyi a common practice is to use `is` for boolean tests, e.g. `is False`, rather than `==`.